Index: Makefile
===================================================================
--- stacks/common/yaml_cpp/Makefile	(revision 33167)
+++ stacks/common/yaml_cpp/Makefile	(working copy)
@@ -12,0 +12,0 @@


 yaml-cpp: $(SOURCE_DIR)/unpacked
-	cd build && cmake -DCMAKE_INSTALL_PREFIX=../yaml-cpp yaml-cpp-0.2.2
+	cd build/yaml-cpp-0.2.2/ ; patch -p1 < ../../yaml-cpp-cmakelist.patch
+	cd build && cmake -DCMAKE_TOOLCHAIN_FILE=`rospack find rosbuild`/rostoolchain.cmake -DCMAKE_INSTALL_PREFIX=../yaml-cpp yaml-cpp-0.2.2
 	cd build && make && make install
 	if [ `uname` = Darwin ]; then \
 		install_name_tool -id `rospack find yaml_cpp`/yaml-cpp/lib/libyaml-cpp.0.2.dylib yaml-cpp/lib/libyaml-cpp.0.2.dylib; \
