--- stacks/common/yaml_cpp/Makefile
+++ stacks/common/yaml_cpp/Makefile
@@ -12,7 +12,8 @@ include $(shell rospack find mk)/download_unpack_build.mk


 yaml-cpp: $(SOURCE_DIR)/unpacked
-	cd build && cmake -DCMAKE_INSTALL_PREFIX=../yaml-cpp yaml-cpp-0.2.2
+	cd build/yaml-cpp-0.2.2/ ; patch -N -p1 < ../../yaml-cpp-cmakelist.patch || :
+	cd build && cmake -DCMAKE_TOOLCHAIN_FILE=`rospack find rosbuild`/rostoolchain.cmake -DCMAKE_INSTALL_PREFIX=../yaml-cpp yaml-cpp-0.2.2
 	cd build && make && make install
 	if [ `uname` = Darwin ]; then \
 		install_name_tool -id `rospack find yaml_cpp`/yaml-cpp/lib/libyaml-cpp.0.2.dylib yaml-cpp/lib/libyaml-cpp.0.2.dylib; \
--
1.7.4.1

